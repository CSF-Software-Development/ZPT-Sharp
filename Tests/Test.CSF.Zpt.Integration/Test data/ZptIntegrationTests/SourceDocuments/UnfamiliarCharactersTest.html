<html>
<body>
<p>This test reproduces the first part of issue https://github.com/csf-dev/ZPT-Sharp/issues/232</p>
<p><span metal:define-macro="my_macro">Here is that unfamiliar character in a macro: ✔</span></p>
<p tal:define="myCharacter string:✔">
  This document contains an unfamiliar character.
  <span metal:use-macro="my_macro">This macro substitution should not trigger an error.</span>
  Also, including the character via TAL shouldn't trigger an error either.
  <span tal:content="myCharacter">The unfamiliar character goes here</span>
</p>
</body>
</html>