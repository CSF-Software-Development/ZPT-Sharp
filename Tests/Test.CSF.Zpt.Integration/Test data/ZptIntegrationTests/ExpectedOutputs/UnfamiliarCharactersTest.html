<html>
<body>
<p>This test reproduces the first part of issue https://github.com/csf-dev/ZPT-Sharp/issues/232</p>
<p><span>Here is that unfamiliar character in a macro: ✔</span></p>
<p>
  This document contains an unfamiliar character.
  <span>Here is that unfamiliar character in a macro: ✔</span>
  Also, including the character via TAL shouldn't trigger an error either.
  <span>&#10004;</span>
</p>
</body>
</html>